packages/analytics-js/__tests__/app/RudderAnalytics.test.ts

```diff
@@ -162,7 +162,7 @@ describe('Core - Rudder Analytics Facade', () => {
     const groupSpy = jest.spyOn(analyticsInstance, 'group');
 
     rudderAnalytics.group(1234);
-    expect(groupSpy).toHaveBeenCalledWith({ groupId: '1234', traits: {} });
```

Comment:
Are we removing root level traits object from the payload for group call if not present?

Reply:
Yes. This is in line with the `identify` API implementation.

Reply:
If customers use this in transformation, that will break things for them.

Reply:
Sounds good